docs/cli/overview.md

```diff
@@ -4,67 +4,67 @@ Kusk is a CLI tool designed to help you manage common tasks related to Kusk Gate
 
 The CLI supports the following commands:
 
-- `install` - installs Kusk Gateway and all its components with a single command - [Read more](install-cmd.md)   
-- `api generate` - generates Kusk Gateway API resources from OpenAPI - [Read more](generate-cmd.md)
+- `install` - Installs Kusk Gateway and all its components with a single command - [Read more](install-cmd.md).  
+- `api generate` - Generates Kusk Gateway API resources from OpenAPI - [Read more](generate-cmd.md).
 
-## Installation
+## **Installation**
 
-### Homebrew
+### **Homebrew**
 
 ```
 brew install kubeshop/kusk/kusk
 ```
 
-### Using golang installation
+### **Using Golang Installation**
 
 ```
 go install github.com/kubeshop/kusk@latest
 ```
 
-To install a particular version: replace `latest` with the version number
+To install a particular version replace `latest` with the version number.
 
-You can get a list of the available kusk versions from our [releases page](https://github.com/kubeshop/kusk/releases)
+You can get a list of the available Kusk versions from our [releases page](https://github.com/kubeshop/kusk/releases).
```

Comment:
```suggestion
You can get a list of the available Kusk Gateway versions from our [releases page](https://github.com/kubeshop/kusk/releases).
```

"Kusk" is a "line of products", "Kusk Gateway" is one of those products, which is what this documentation is all about.